src/GHC/Util.hs

```diff
@@ -49,8 +49,25 @@ fakeSettings = Settings
 fakeLlvmConfig :: (LlvmTargets, LlvmPasses)
 fakeLlvmConfig = ([], [])
 
+badExtensions :: [Extension]
+badExtensions =
+  [
+    Arrows
+  , TransformListComp
+  , UnboxedTuples
+  , UnboxedSums
+  , QuasiQuotes
+  , RecursiveDo
+ ]
+
 enabledExtensions :: [Extension]
-enabledExtensions = [Cpp .. StarIsType] -- First and last extension in ghc-boot-th/GHC/LanguageExtensions/Type.hs 'data Extension'.
+enabledExtensions =
+  -- `Cpp`/`StarIsType` are the first and last extension in
+  -- ghc-boot-th/GHC/LanguageExtensions/Type.hs 'data Extension'. MR
+  -- https://gitlab.haskell.org/ghc/ghc/merge_requests/826 has landed
```

Comment:
This has now landed. Can we switch to minBound/maxBound?

Reply:
I think not because we require `hlint` compile with 8.4 right?

Reply:
Argh, of course you are right. I get so confused with this compiler in the compiler :) I'll give it a go.